src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -41,7 +41,7 @@
   /**
    * Draws the route to the screen.
    */
-  public void drawRoute(final Graphics2D graphics, final RouteDescription routeDescription, final MapPanel view,
+  public void drawRoute(final Graphics2D graphics, final RouteDescription routeDescription, final MapPanel mapPanel,
```

Comment:
Long method code smell, evident in the deep nested logic statements and the need to declare so many variables up front (such as xOffset, yOffset). It's okay'ish as is, but could use some cleanup/decomposition.